src/main/java/blusunrize/immersiveengineering/common/blocks/generic/PoweredMultiblockTileEntity.java

```diff
@@ -137,10 +138,7 @@ public void writeCustomNBT(CompoundNBT nbt, boolean descPacket)
 
 	protected CompoundNBT writeProcessToNBT(MultiblockProcess process)
 	{
-		CompoundNBT tag = process.recipe.writeToNBT(new CompoundNBT());
-		tag.putInt("process_processTick", process.processTick);
-		process.writeExtraDataToNBT(tag);
-		return tag;
+		throw new UnsupportedOperationException();
```

Comment:
This should be changed to write the ID of the recipe, and the same for `readRecipeFromNBT`.